pages/common/gdown.md

```diff
@@ -0,0 +1,24 @@
+# gdown
+
+> Download files from Google Drive and other URLs.
+> More information: <https://github.com/wkentaro/gdown>.
+
+- Download a file from a URL:
+
+`gdown {{url}}`
+
+- Download using a file ID:
+
+`gdown {{file_id}}`
+
+- Download with fuzzy file ID extraction (also works with <https://docs.google.com> links):
+
+`gdown --fuzzy {{url}}`
+
+- Download a folder using its ID or the full URL:
+
+`gdown {{folder_id|url}} -O {{path/to/output_directory}} --folder`
+
+- Write stdout and pipe to extract:
+
+`gdown {{tar_url}} -O - --quiet | tar zxvf -`
```

Comment:
`tar` will determine the compression method automatically. If you use the `z` option and try to extract a tarball compressed with something other than `gzip`, you'll get an error instead.
```suggestion
- Download a tar archive, write it to `stdout` and extract it:

`gdown {{tar_url}} -O - --quiet | tar xvf -`
```